athena-jdbc/src/main/java/com/amazonaws/athena/connectors/jdbc/connection/JdbcConnectionFactory.java

```diff
@@ -33,26 +33,4 @@
      * @return JDBC connection. See {@link Connection}.
      */
     Connection getConnection(JdbcCredentialProvider jdbcCredentialProvider);
-
-    /**
-     * Databases supported to create JDBC connection.
-     */
-    enum DatabaseEngine
```

Comment:
If this was a public enum in a standalone file, wouldn't other modules be able to reference this?

Reply:
The question then becomes where we'd put it? It would be weird to have it in `athena-jdbc`, that would be a cyclic reference.